Code/Engine/Foundation/Platform/Posix/OSFile_Posix.inl

```
@@ -59,6 +59,21 @@ ezResult ezOSFile::InternalOpen(ezStringView sFile, ezFileOpenMode::Enum OpenMod
     return EZ_FAILURE;
   }
 
+  struct stat stats = {};
+  if (fstat(fd, &stats) != 0)
+  {
+    ezLog::Error("Failed to stat file {}, error {}", szFile, ezArgErrno(errno));
```

Such low level functions shouldn't write to the log.

---

Good point, I wasn't sure whether I should log here. There are a few log messages in this file and I'm not sure these should be in here either.

---

Probably not. Most file system functions can fail under normal circumstances. They should just return an error and leave the logging to higher level code.

---

Ok, I'll remove them then. There also aren't any equivalent log messages in the Windows implementation.